### ISSUE TITLE
Keep dark mode setting in the (Redux) state

### ISSUE THREAD

**Issue by joostdecock** [freesewing/website#48]
The [Base layout component](https://github.com/freesewing/website/blob/develop/src/components/layouts/Base.js) keeps a `dark` property in its state that controls the website's dark mode.

This seems like a good candidate to add to the store, so that dark mode is preserved across page loads.

Extra bonus points for synchronizing state with local storage so that it's preserved across browser sessions.

**Comment by joostdecock** [comment]
Hey @sophiekoonin , would this be a good candidate to implement a reducer?

There's no API calls involved, it's just an application-wide setting. So I was wondering if you felt like it, could you implement it, and then we can use your example for more involved stuff :)

**Comment by sophiekoonin** [comment]
Yes, I think that would make sense. I can have a go at that for you tomorrow.
